blueocean-pipeline-editor/src/main/less/accordion.less

```diff
@@ -0,0 +1,102 @@
+.sheet-body .Accordion {
+  position: absolute;
+  top: -20px;
+  right: -32px;
+  bottom: -32px;
+  left: -32px;
+  background: #eef1f3;
+
+  > .Label {
+    background: #f8f8f8;
+    padding: 0 32px;
+  }
+
+  > .Content {
+    padding: 0 32px;
+
+    > *:first-child {
+      margin-top: 32px;
+    }
+
+    > *:last-child {
+      margin-bottom: 32px;
+    }
+  }
+}
+
+.Accordion {
+  @transition-time: 0.4s;
+
+  display: flex;
+  -webkit-flex-direction: column;
+  -moz-flex-direction: column;
+  -ms-flex-direction: column;
+  flex-direction: column;
+  -webkit-flex-flow: column;
+  -moz-flex-flow: column;
+  -ms-flex-flow: column;
+  flex-flow: column;
```

Comment:
Are the prefixes for `flow-direction` and `flex-flow` actually required? I haven't been using them, although maybe that's introduced subtle bugs elsewhere.